### DISCUSSION TITLE
How to get the name of the current filetype icon ?

### DISCUSSION THREAD

**Discussion by radvil** [kyazdani42/nvim-web-devicons#400] [Q&A]
```lua
      ["angular.json"] = {
        icon = "",
        color = "#f38ba8",
        name = "AngularJson",
      }
```

Above is my custom config for overrides. I want to  be able to get the name of the current icon name which is `AngularJson` in this case. How to do this ?

**Comment by alex-courtis** [comment]
Not quite sure it's what you're looking for, but you can use [get_icons](https://github.com/nvim-tree/nvim-web-devicons?tab=readme-ov-file#get-all-icons) to retrieve everything.

If you want just one you could call `get_icon` and extract the name from the highlight group e.g. `DevIconAngularJson`
